README.md

```diff
@@ -56,49 +56,3 @@ $ docker build . -t libxmtp:1
 - [`bindings_ffi`](https://github.com/xmtp/libxmtp/tree/main/bindings_ffi): FFI bindings for Android and iOS (in progress)
 - [`bindings_js`](https://github.com/xmtp/libxmtp/tree/main/bindings_js): JS bindings (in progress)
 - [`bindings_wasm`](https://github.com/xmtp/libxmtp/tree/main/bindings_wasm): Wasm bindings (in progress)
-
-## Key features of XMTP v3-alpha
-
-### Double ratchet messaging
-
-**What**: XMTP v3-alpha introduces the use of the [double ratchet algorithm](https://signal.org/docs/specifications/doubleratchet/) to keep past and future messages secure.
-
-**Why**: With double ratchet messaging, each message is encrypted by its own key. If a message key becomes compromised:
-
-- All of their past messages remain secure. (Forward secrecy)
-
-  Specifically, a malicious actor can’t decrypt and read the user’s past messages.
-
-- Future messages will be secure. (Future, or post-compromise, secrecy)
-
-  Specifically, a malicious actor can’t decrypt and read future messages. They also can’t send new messages impersonating the user.
-
-In contrast, with XMTP v2, if a user’s key bundle becomes compromised:
-
-- All of their past messages might not remain secure.
-- Future messages might not remain secure.
-
-**How**: Specifically, it uses the Rust version of [Olm](https://gitlab.matrix.org/matrix-org/olm/-/blob/master/docs/olm.md), which is Matrix protocol’s implementation of double ratchet. To learn more about the double ratchet algorithm and how sessions and session keys work, see [The Double Ratchet Algorithm](https://signal.org/docs/specifications/doubleratchet/).
-
-> **Tip**  
-> Use the [example CLI](https://github.com/xmtp/libxmtp/blob/main/examples/cli/README.md) to try out sending XMTP v3-alpha double ratchet messages on the XMTP `dev` network.
-
-### Installation key bundles
```

Comment:
This section is basically still accurate, but we cover it in the new credentials doc anyways.